java/vector/src/main/java/org/apache/arrow/vector/ipc/ArrowReader.java

```diff
@@ -229,4 +242,18 @@ protected void loadDictionary(ArrowDictionaryBatch dictionaryBatch) {
       dictionaryBatch.close();
     }
   }
+
+  /**
+   * Concat dictionary vector and delta dictionary vector.
+   */
+  private void concatDeltaDictionary(FieldVector vector, FieldVector deltaVector) {
```

Comment:
lets open up a JIRA to remove this method once https://github.com/apache/arrow/pull/5916 is merged.

Reply:
right, I will follow this issue.